wai-extra/README.md

```diff
@@ -1,3 +1,16 @@
-## wai-extra
+# wai-extra
 
 The goal here is to provide common features without many dependencies.
+
+
+## Example using Server-Sent Events ##
+
+There is a small example using Server-Sent Events (SSE) in the
+`./example` directory.
+
+Run the commands below to start the server on http://localhost:8080
+
+```
+$ cabal configure -f build-example
```

Comment:
Let's stick to Stack recommendations here, which will deal with local packages and not drop people into dependency solving problems.

    stack build .
    stack exec example